packages/world/src/modules/core/CoreModule.sol

```diff
@@ -81,11 +85,9 @@ contract CoreModule is IModule, WorldContextConsumer {
       msgSender: _msgSender(),
       msgValue: 0,
       target: coreSystem,
-      funcSelectorAndArgs: abi.encodeWithSelector(
-        WorldRegistrationSystem.registerSystem.selector,
-        ResourceSelector.from(ROOT_NAMESPACE, CORE_SYSTEM_NAME),
-        coreSystem,
-        true
+      funcSelectorAndArgs: abi.encodeCall(
```

Comment:
not blocking this PR but would it make more sense to call this `calldata`?

Reply:
we use `funcSelectorAndArgs` in a couple places, could consider changing that to `callData` (`calldata` wouldn't work because it's a reserved word). Let's track in an issue!

Reply:
https://github.com/latticexyz/mud/issues/1460